README.md

```diff
@@ -185,6 +145,8 @@ You may need to configure Coverband you can either do that passing in all config
 ```ruby
 #config/coverband.rb
 Coverband.configure do |config|
+  config.store = Coverband::Adapters::RedisStore.new(Redis.new(url: ENV['MY_REDIS_URL']))
```

Comment:
Added some more typical configuration settings since now s3 is something that will most often not be needed.

Reply:
makes sense